### ISSUE TITLE
`astro-i18n-aut` upgrade

### ISSUE THREAD

**Issue by jlarmstrongiv** [dallyh/website-concept#1]
Just two features I thought you’d like in the latest version:

```ts
import {
  locales,
  localeKeys,
  defaultLocale,
  getLocale,
  getAllLocaleUrls,
  getLocaleUrl,
  getUrlWithoutLocale,
} from "astro-i18n-aut"
```

`getLocales` is no longer needed, since you can now import `defaultLocale`, `locales`, and `localeKeys` from the package directly

You no longer need `getLocaleEx` since `getLocale` now returns the `defaultLocale`

Anyway, it’s nice to see others use the `astro-i18n-aut` so I can see what changes would be helpful 😄

**Comment by dallyh** [comment]
@jlarmstrongiv Amazing! Thanks for this. I didn't find this info in your docs, maybe adding it as a section of what you can import could help some other people too 😁

**Comment by jlarmstrongiv** [comment]
@dallyh Planning to 😄 I’m waiting on another core astro bug to see why my virtual modules aren’t refreshing when the config changes.  They have their hands pretty full with the v3 release at the moment though
